<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2183</width>
    <height>1074</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background : #444444  ;color : rgb(255, 255, 255)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="1" column="1" colspan="3">
    <widget class="QLabel" name="msgLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>MSG</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="Line" name="line">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>5000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Choose a subject</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="showHistoryButton">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Show History</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboProject">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="delete_subject_Button">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Delete This Subject</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="delete_project_Button">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Delete This Project</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboSubject">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________________</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="email_history_Button">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Email History</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Choose a project</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="before_days">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>90</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>All</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="0" colspan="4">
      <widget class="QTableWidget" name="tableWidget">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background : rgb(152, 152, 152)  ;color :  rgb(74, 74, 74)</string>
       </property>
       <column>
        <property name="text">
         <string>name_of_session</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>done</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>task</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>task_done</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>normalTime</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>start_time</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>finish_time</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>time_consumed</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>date</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>In the last .. days :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="3">
      <widget class="QPushButton" name="start_promodoro_Button">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Start Promodoro</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QLineEdit" name="subjectText">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="addSubjecttButton">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Add New Subject</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="comboProject2">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Adding New Subject :</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="1">
      <widget class="QLineEdit" name="projectText">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Adding New Project : </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>back</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="addProjectButton">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Add New Project</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QPushButton" name="add_r_Button">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Add New Recipient</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="emails_combo">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="recipient_input">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="delete_r_Button">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Delete This recipient</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Adding/ deleting recipients :</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
